libevmasm/SemanticInformation.cpp

```diff
@@ -35,6 +35,7 @@ bool SemanticInformation::breaksCSEAnalysisBlock(AssemblyItem const& _item)
 	default:
 	case UndefinedItem:
 	case Tag:
+	case PushDeployTimeAddress:
```

Comment:
Does this break `const`ness or `static`ness?

Reply:
Hm, I don't think so, but I will check again.

Reply:
Ok, checked it: I don't think this even applies here. We only have `const`ness and `static`ness for instructions/operations, but `PushDeployTimeAddress` is not an operation.